### PR TITLE
fix for issue #72

### PR DESCRIPTION
**Pull request by Astromations**
two commits cause I accidentally omitted a line and misnamed the issue.

### CODE REVIEW

**Comment by nimsandu** [comment]
@Astromations Just to clarify your fix ill request @kyrie25 to review this fix ok?

**Comment by nimsandu** [comment]
@kyrie25 can you share your opinion regarding this pr before merge

**Comment by kyrie25** [comment]
> @kyrie25 can you share your opinion regarding this pr before merge 

Have not tested but making the icon disappear doesn't really count as fixing the issue but rather a lazy (no offense) workaround to omit a Spotify feature, of which will raise questions among users.
Would be nice if you can find a way to realign the icons instead.

**Comment by nimsandu** [comment]
@Astromations Can you work regarding @kyrie25 s review please?

@kyrie25 Thank you for sharing your idea

**Comment by MGPlayzYT** [comment]
It could be fixed the same way as I did in #71.

**Comment by Astromations** [comment]
> > @kyrie25 can you share your opinion regarding this pr before merge
> 
> Have not tested but making the icon disappear doesn't really count as fixing the issue but rather a lazy (no offense) workaround to omit a Spotify feature, of which will raise questions among users. Would be nice if you can find a way to realign the icons instead.

If I did keep the icon it'd prevent users from clicking on the center of the playlist.

**Comment by kyrie25** [comment]
The icon is marginally small, plus that problem does not arise from the users since it's not the only place they can click to select the playlist.
Removing the feature on the other hand, would arise among users.

**Comment by MGPlayzYT** [comment]
> > > @kyrie25 can you share your opinion regarding this pr before merge
> > 
> > 
> > Have not tested but making the icon disappear doesn't really count as fixing the issue but rather a lazy (no offense) workaround to omit a Spotify feature, of which will raise questions among users. Would be nice if you can find a way to realign the icons instead.
> 
> If I did keep the icon it'd prevent users from clicking on the center of the playlist.

It's the same case with the playing icon.

**Comment by Astromations** [comment]
> The icon is marginally small (taking up less than 1/9th of the playlist icon), plus that problem does not arise from the users since it's not the only place they can click to select the playlist. Removing the feature on the other hand, would arise questions among users.

So should I keep the icon and just resize the playlist image

File: user.css [hunk 1]
```diff
@@ -811,6 +811,10 @@ button.switch {
   color: var(--spice-shadow);
 }
 
+.main-rootlist-statusIcons {
+  display: none !important;
+}
+
```
Comment (kyrie25) [inline user.css:817]:
```suggestion
.main-rootlist-statusIcons > div {
  margin-left: 8px
}

```

**Review by kyrie25 — CHANGES_REQUESTED** [review]
This will suffice.

**Comment by Astromations** [comment]
> This will suffice, as it'd only realign the icon and nothing more. Had some free time so I looked into the issue.



> This will suffice, as it'd only realign the icon and nothing more. Had some free time so I looked into the issue.

That works. I'll keep the icon removed on my end though cause I find it kind of ugly 
image

**Comment by nimsandu** [comment]
@kyrie25 @Astromations  so we are ready to marge right?

**Comment by kyrie25** [comment]
All good.

**Comment by nimsandu** [comment]
copy that,
merging now

**Comment by nimsandu** [comment]
@Astromations @kyrie25 @MGPlayzYT thank you

**Pull request merged**